apps/codeforafrica/src/components/PageHeader/PageHeader.js

```diff
@@ -0,0 +1,35 @@
+import { Section, RichTypography } from "@commons-ui/core";
+import { Box } from "@mui/material";
+import React from "react";
+
+import TwoToneBackground from "../TwoToneBackground";
+
+const PageHeader = React.forwardRef(function PageHeader(props, ref) {
+  const { title, description } = props;
+  if (!title && !description) {
+    return null;
+  }
+  return (
+    <Box ref={ref} sx={{ backgroundColor: "background.main" }}>
+      <Section>
+        <TwoToneBackground>
```

Comment:
`TwoToneBackground` is supposed to cover end to end i.e. should be where `Box` is.